src/index.html

```diff
@@ -11,8 +11,61 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+        src="images/imac.jpeg"
+        alt="Imac laptop by Apple"
+        class="card__content"
+      />
+
+      <span class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </span>
+
+      <span class="card__id">Product code: 195434</span>
+
+      <div class="card__reviews">
+        <ul class="stars">
+          <li class="stars__star stars__star--1"></li>
+          <li class="stars__star stars__star--2"></li>
+          <li class="stars__star stars__star--3"></li>
+          <li class="stars__star stars__star--4"></li>
+          <li class="stars__star stars__star--5"></li>
```

Comment:
```suggestion
        <ul class="stars stars--3">
          <li class="stars__star></li>
          <li class="stars__star"></li>
          <li class="stars__star"></li>
          <li class="stars__star"></li>
          <li class="stars__star"></li>
```
It should work like this